### PR TITLE
Add new custom pg exporter config

### PR DESCRIPTION
**Pull request by Kidswiss**
## Summary

StackGres' default exporter configuration doesn't play well with large databases.

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [x] Update tests.
- [ ] Link this PR to related issues.

### CODE REVIEW

File: pkg/comp-functions/functions/vshnpostgres/pgqexporter_config.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+/*
+	This code ensures we have minimalistic postgresql-exporter configuration
+	It prevents high memory usage for big databases
+*/
+//go:embed scripts/queries.yml
+var queries string
+
+func PgExporterConfig(ctx context.Context, comp *vshnv1.VSHNPostgreSQL, svc *runtime.ServiceRuntime) *v1beta1.Result {
+	err := svc.GetObservedComposite(comp)
```
Comment (zugao) [inline pkg/comp-functions/functions/vshnpostgres/pgqexporter_config.go:24]: Do you disable somewhere the current exporter?

File: pkg/comp-functions/functions/vshnpostgres/scripts/queries.yml [hunk 2]
```diff
@@ -0,0 +1,880 @@
+pg_replication:
```
Comment (zugao) [inline pkg/comp-functions/functions/vshnpostgres/scripts/queries.yml:1]: Where is this coming from? How can we ensure this is updated in time?

File: pkg/comp-functions/functions/vshnpostgres/pgqexporter_config.go (hunk 1 above)
Comment (Kidswiss) [reply pkg/comp-functions/functions/vshnpostgres/pgqexporter_config.go:24]: We don't disable anything, we just override the default configs from stackgres.

File: pkg/comp-functions/functions/vshnpostgres/scripts/queries.yml (hunk 2 above)
Comment (Kidswiss) [reply pkg/comp-functions/functions/vshnpostgres/scripts/queries.yml:1]: This is from the original stackgres exporter config.

I did a `curl localhost:9187/metrics | grep -o ".*{" | uniq -c | sort -bgr` on the container to identify all the really big metrics and just removed them from the configs.

As for updating them... We could probably observe the original confing in the compfunction and remove the broken queries then dynamically 🤔

**Review by zugao — APPROVED** [review]
LGTM

**Pull request merged**